phoenix-core/src/main/java/org/apache/phoenix/schema/PSchemaKey.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Where is this used? I don't see any mention of PSchemaKey outside this class itself.